public/app/features/alerting/unified/components/contact-points/__snapshots__/useContactPoints.test.tsx.snap

```diff
@@ -39,7 +39,6 @@ exports[`useContactPoints should return contact points with status 1`] = `
           },
         },
       ],
-      "provisioned": false,
```

Comment:
now, in this test, the data is from the alert manager config , where we don't have this field